vendor/processors/authorizenet.py

```diff
@@ -29,6 +29,19 @@
 from .base import PaymentProcessorBase
 
 
+
+class customdate(pyxb.binding.datatypes.date):
```

Comment:
Classes are typically Camel Case.  Any issue with this being lowercase?

Reply:
Agree will change it to Camel Case